prdoc/pr_7569.prdoc

```diff
@@ -0,0 +1,11 @@
+title: 'slot-based-collator: Allow multiple blocks per slot'
+doc:
+- audience: Node Operator
+  description: Adds multiple blocks per slot support to the slot-based collator
```

Comment:
We need to mention here that we are deprecating cli parameters and introducing the authoring style params.